### PR TITLE
Correct Badly Formatted JSON

### PR DESCRIPTION
**Pull request by apeschel**
### Contributing guide

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md
### Changelog
- Correct badly formatted JSON
### Description of the change

This commit fixes several JSON files with invalid JSON.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@apeschel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: 201-decrypt-running-windows-vm/azuredeploy.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "extensionName": "AzureDiskEncryption",
     "extensionVersion": "1.1",
     "encryptionOperation": "DisableEncryption",
-    "updateEncryptionSettingsUrl": "[concat('https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-decrypt-running-windows-vm/updateEncryptionSettings-',parameters('volumeType'),'.json')]",	
+    "updateEncryptionSettingsUrl": "[concat('https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-decrypt-running-windows-vm/updateEncryptionSettings-',parameters('volumeType'),'.json')]"
```
Comment (krnese) [inline 201-decrypt-running-windows-vm/azuredeploy.json]: add a parameter for _artifactsLocation and _artifactsLocationSasToken according to https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc .
Then in a variable, concat to something similar as:
"url": "[concat(parameters('_artifactsLocation'), '/nestedtemplates/templatename.json', parameters('_artifactsLocationSasToken'))]"

**Comment by gatneil** [comment]
Hey there! Sorry for the delay; can you sign the CLA? I can't merge this in otherwise :(

**Comment by gatneil** [comment]
No activity; closing

**Pull request closed without merging**